### PR TITLE
GUACAMOLE-135: Add Swiss-French keymap for RDP

### PR DESCRIPTION
**Pull request by jmroth**

### CODE REVIEW

**Comment by mike-jumper** [comment]
Thanks, @jmroth.

Can you rebase to combine these commits (or reword such that the commit message matches the specific high-level change made by that commit)? There shouldn't be git history with repeated identical commit messages.

File: src/protocols/rdp/keymaps/fr_ch_qwertz.keymap [hunk 1]
```diff
@@ -0,0 +1,45 @@
+parent  "base"
```
Comment (mike-jumper) [inline src/protocols/rdp/keymaps/fr_ch_qwertz.keymap]: The license which applies to this file needs to be explicitly stated. See the other keymaps:

https://github.com/jmroth/incubator-guacamole-server/blob/219ce42c3338357dd4fff9dda2944edbdf98af73/src/protocols/rdp/keymaps/de_de_qwertz.keymap#L1-L19

**Comment by mike-jumper** [comment]
I had originally thought you were @Robin-73, given the other almost-identical pull request related to GUACAMOLE-135. Any opinion on your changes vs. the pending changes in #33?

**Comment by jmroth** [comment]
Uh, didn't know there were other incoming contribs concerning this matter. I can try to rebase. Not a git expert though. Simply used the github GUI. What's the problem with #33? Is it not being accepted then?

On January 28, 2017 8:44:06 PM CET, Mike Jumper <notifications@github.com> wrote:
>I had originally thought you were @Robin-73, given the other
>almost-identical pull request related to GUACAMOLE-135. Any opinion on
>your changes vs. the pending changes in #33?
>
>-- 
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/apache/incubator-guacamole-server/pull/54#issuecomment-275869923

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by mike-jumper** [comment]
The only issue with #33 is that it's still not rebased such that the commits are up to spec (tagged with JIRA issue, message contains high-level description of change). Right now, most of the commits say things like "Create fr_ch_qwertz.keymap" and "Update fr_ch_qwertz.keymap", which we can't accept (git already tells us that much automatically).

**Comment by Robin-73** [comment]
hi,
i still did not get then time to cleanup and rebase my contrib,
not a github expert as well, so still need to read and learn how to rebase.

regards,
robin-73
________________________________
De : Mike Jumper [notifications@github.com]
Date d'envoi : samedi 28 janvier 2017 22:15
À : apache/incubator-guacamole-server
Cc : Cédric Marie-Marthe; Mention
Objet : Re: [apache/incubator-guacamole-server] GUACAMOLE-135: Add Swiss-French keymap for RDP (#54)


The only issue with #33<https://github.com/apache/incubator-guacamole-server/pull/33> is that it's still not rebased such that the commits are up to spec (tagged with JIRA issue, message contains high-level description of change). Right now, most of the commits say things like "Create fr_ch_qwertz.keymap" and "Update fr_ch_qwertz.keymap", which we can't accept (git already tells us that much automatically).

**Comment by mike-jumper** [comment]
@jmroth and @Robin-73, how about:

1. I'll take the changes from #33 and rebase them on @Robin-73's behalf.
2. I'll open a new PR for those changes (they will show one commit from @Robin-73).
3. Since I'll have opened the PR, someone else on the project will review it, and hopefully things move forward then.

Sound reasonable? Having two "competing" PRs for the same exact change, both in need of rebase, is definitely a new situation.

**Comment by Robin-73** [comment]
that's would be perfect !
thanks you.
________________________________
De : Mike Jumper [notifications@github.com]
Date d'envoi : samedi 28 janvier 2017 22:25
À : apache/incubator-guacamole-server
Cc : Cédric Marie-Marthe; Mention
Objet : Re: [apache/incubator-guacamole-server] GUACAMOLE-135: Add Swiss-French keymap for RDP (#54)


@jmroth<https://github.com/jmroth> and @Robin-73<https://github.com/Robin-73>, how about:

  1.  I'll take the changes from #33<https://github.com/apache/incubator-guacamole-server/pull/33> and rebase them on @Robin-73<https://github.com/Robin-73>'s behalf.
  2.  I'll open a new PR for those changes (they will show one commit from @Robin-73<https://github.com/Robin-73>).
  3.  Since I'll have opened the PR, someone else on the project will review it, and hopefully things move forward then.

Sound reasonable? Having two "competing" PRs for the same exact change, both in need of rebase, is definitely a new situation.

**Pull request closed without merging**